eng/LicenseHeader.txt

```diff
@@ -0,0 +1,4 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
```

Comment:
cc @Eilon  - I think aspnet is actually under the Apache 2.0 license.

Reply:
Whoops, copied from corefx